website/www/site/content/en/documentation/io/managed-io.md

```diff
@@ -180,6 +184,17 @@ manual updates or user intervention required!)
         Properties passed to the Hadoop Configuration.
       </td>
     </tr>
+    <tr>
+      <td>
+        drop
+      </td>
+      <td>
+        <code>list[<span style="color: green;">str</span>]</code>
+      </td>
```

Comment:
do we need to specify the required Beam SDK version?

Reply:
Hmmm not sure how to do this considering the file is auto-generated. Maybe we can include it in the schema field's description but that doesn't seem very clean

Reply:
Won't it work, even on SDKs that don't understand it yet?

Reply:
only on Dataflow Runner V2 -- it'll fail if a user tries experimenting with any other runner + old SDK